preferences/legacy/kustomization.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ../components/interfacemodel-e1000e
+  - ../components/diskbus-scsi
+
+patches:
+  - target:
+      kind: VirtualMachinePreference
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: legacy
+  - target:
+      kind: VirtualMachineClusterPreference
+    patch: |-
+      - op: replace
+        path: /metadata/name
+        value: legacy
+  - target:
+      kind: VirtualMachinePreference
+    patch: |-
+      - op: replace
+        path: /metadata/labels/instancetype.kubevirt.io~1os-type
```

Comment:
What's the reason for adding this and the other metadata below?

Reply:
Due to the fact that the legacy preference uses `base`, these annotations and labels are added as well with value `linux`. I just thought that this may be confusing for users, so I adjusted with these patches to `legacy`. Nevertheless, maybe it is better just to drop them.
On the other hand, I've found very convinient using the preference `base`

Reply:
Right sorry I forgot that `base` did that, can you move the metadata changes into `preference/legacy/metadata/metadata.yaml` as we do for Windows (see `preferences/windows/base/metadata/metadata.yaml`)? I don't mind keeping the spec changes in here but if we need anymore it would be better to also break that out into a standalone file.

Reply:
Can you also update `docs/labels.md` with the new values of `os-type`?

Reply:
All done! Excellent feedback, much appreciated :)